registration-processor/registration-processor-info-storage-service/src/main/java/io/mosip/registration/processor/packet/storage/entity/RegBioRefEntity.java

```diff
@@ -33,6 +33,12 @@
 	@Column(name = "upd_dtimes")
 	private LocalDateTime updDtimes;
 
+	@Column(name = "reg_type")
```

Comment:
Is it possible to add these 2 properties to the RegBioRefPKEntity class rather ?

Reply:
yes.
but we have change the primary key of table also.

Reply:
Need to change the primary key of table. I did the coding chnages.

Reply:
Remove the unwanted line RegBioRefEntity.java